app/controllers/api/v1/ecf_participants_controller.rb

```diff
@@ -41,19 +41,41 @@ def lead_provider
       end
 
       def participants
-        participants = lead_provider.ecf_participants
-                                    .distinct
-                                    .includes(
-                                      teacher_profile: {
-                                        current_ecf_profile: %i[cohort school ecf_participant_eligibility ecf_participant_validation_data participant_profile_state participant_profile_states schedule],
-                                        early_career_teacher_profile: :mentor,
-                                      },
-                                    )
-                                    .where(school_cohorts: { cohort_id: Cohort.current.id })
-
-        participants = participants.changed_since(updated_since) if updated_since.present?
-
-        participants.order(:created_at)
+        participant_profiles = ParticipantProfile::ECF.where(id: participant_profile_ids)
+                                                      .joins(:user)
+                                                      .includes(
+                                                        :user,
+                                                        :cohort,
+                                                        :school,
+                                                        :ecf_participant_eligibility,
+                                                        :ecf_participant_validation_data,
+                                                        :schedule,
+                                                        teacher_profile: { current_ect_profile: { mentor_profile: :user } },
+                                                      )
+
+        if updated_since.present?
+          participant_profiles = participant_profiles.where(user: { updated_at: updated_since.. })
```

Comment:
What's this `updated_since..` syntax? Never seen this? 
is this a shorthand for something in which case could we use a more idiomatic syntax?

Reply:
also I believe the order didn't work because this probably should be:

```ruby
.where(users: { updated_at: updated_since.. })
.order("users.updated_at")
`

Reply:
no need to order by `users.id` since those `id` are `UUID` and the order of those is pretty random

Reply:
It's an endless range — it means updated from that time. 

Re: ordering. That's how it works at the moment so let's keep it consistent for now.